_includes/head.html

```diff
@@ -13,6 +13,9 @@
     <!-- Nav infrastructure and misc utils (search, user cards, embeds, CC license) -->
     <link rel="stylesheet" href="{{ site.github.url }}/stylesheets/infrastructure-utils.css" />
 
+    <!-- Python code highlighting -->
```

Comment:
That's exactly what I used. But the python css for that was missing, so I added it.